source/catalog.html

```diff
@@ -0,0 +1,258 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Catalog</title>
+  </head>
+  <body>
+    <header class="main-header">
+      <a class="main-header__logo" href="#">
+        <img class="main-header__logo-image" src="#" alt="#" />
+      </a>
+      <a class="main-header__title" href="#">
+        <img class="main-header__title-logo" src="#" alt="#" />
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню.</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav-list">
+            <li class="main-nav-list__item">
+              <a href="index.html">главная</a>
+            </li>
+            <li class="main-nav-list__item main-nav-list__item--current">
+              <a href="#">каталог продукции</a>
+            </li>
+            <li class="main-nav-list__item">
+              <a href="form.html">подбор программы</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="catalog-container">
+      <section class="catalog">
+        <h1 class="catalog__title">Каталог продукции</h1>
+        <ul class="catalog-list">
+          <li class="catalog-list__item">
+            <h2 class="catalog-list__title">Cat Energy PRO 500 г</h2>
```

Comment:
```suggestion
            <h2 class="catalog-list__item-title">Cat Energy PRO 500 г</h2>
```
а может вот так писать надо? найди парочку известных сайтов по БЭМу и посмотри их практику
как они списки оформляют?